commands/meshuggah.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function meshuggah (controller) {
+  // Serve up a Meshuggah video
+  controller.hears('Meshuggah', params.triggerEvents, function(bot,message) {
+    Scrape.request('https://www.youtube.com/user/Meshuggah', function (err, $) {
```

Comment:
`Scrape` is not defined. This would need to be imported and included. If you want to use the Youtube API, it would be best to use something like axois and make a request to the youtube api to get the data, not just scrape the webpage. In fact, you learn how to do this today!!